packages/schematics/angular/application/index.ts

```diff
@@ -368,7 +368,7 @@ export default function (options: ApplicationOptions): Rule {
           }),
           move(sourceDir),
         ]), MergeStrategy.Overwrite),
-      schematic('e2e', e2eOptions),
+      !options.minimal ? schematic('e2e', e2eOptions) : noop(),
```

Comment:
can you invert the logic and remove the negation?

Reply:
Sure! I tough it was a kind of convention to use noop as "else" :sweat_smile: